Dockerfile.dapper

```diff
@@ -42,6 +42,9 @@ ENV DAPPER_OUTPUT ./bin ./dist
 ENV DAPPER_DOCKER_SOCKET true
 ENV DAPPER_RUN_ARGS "-v /run/containerd/containerd.sock:/run/containerd/containerd.sock -v harvester-installer-go:/root/go -v harvester-installer-cache:/root/.cache"
 
+# You cloud defined your own rke2 url to make iso building fast
+ENV DAPPER_ENV RKE2_IMAGE_REPO=
```

Comment:
I suggested add the variable just to line 39
```
ENV DAPPER_ENV REPO TAG DRONE_TAG DRONE_BRANCH CROSS RKE2_IMAGE_REPO
```

Reply:
Done. please review again